include/dxc/Support/SPIRVOptions.h

```diff
@@ -1,24 +1,25 @@
-//===-- EmitSPIRVOptions.h - Options for SPIR-V CodeGen ---------*- C++ -*-===//
-//
-//                     The LLVM Compiler Infrastructure
-//
-// This file is distributed under the University of Illinois Open Source
-// License. See LICENSE.TXT for details.
-//
-//===----------------------------------------------------------------------===//
-#ifndef LLVM_CLANG_SPIRV_EMITSPIRVOPTIONS_H
-#define LLVM_CLANG_SPIRV_EMITSPIRVOPTIONS_H
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+// SPIRVOptions.h                                                            //
```

Comment:
We should use LLVM copyright header here.

Reply:
Done